core/foundation/src/TClassEdit.cxx

```diff
@@ -1215,14 +1231,13 @@ int TClassEdit::GetSplit(const char *type, vector<string>& output, int &nestedLo
 string TClassEdit::CleanType(const char *typeDesc, int mode, const char **tail)
 {
    static const char* remove[] = {"class","const","volatile",0};
-   static bool isinit = false;
-   static std::vector<size_t> lengths;
-   if (!isinit) {
+   static std::vector<size_t> lengths{ []() {
```

Comment:
I find lamda-in-intialization really hard to read. Would it be reasonable to split the lambda into a dedicated declaration?